standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveAlterHandler.java

```diff
@@ -1112,6 +1081,32 @@ public static List<ColumnStatistics> alterTableUpdateTableColumnStats(RawStore m
     return newMultiColStats;
   }
 
+  @VisibleForTesting
+  public static void updateTableColumnStats(RawStore msdb, Table newTable, String validWriteIds, List<ColumnStatistics> columnStatistics)
```

Comment:
is there a helper class handling the stats? i would extract these methods there

Reply:
According to our offline conversation, I file a new Jira for creating a new stats related utility class, and move all realted methods there. For now only the static modifier is removed.